content/06.know-your-problem.md

```diff
@@ -3,29 +3,29 @@
 In order to ensure similar distributions of important characteristics across study groups in the absence of randomization, individuals may be matched based on age, gender, or weight.
 Study designs will often have different assumptions and caveats, and these cannot be ignored during a data analysis.
 Many datasets are now passively collected or do not have a specific design, but even in this case it is important to know how individuals or samples were treated.
 Samples originating from the same study site, oversampling of ethnic groups or zip codes, and sample processing differences are all sources of variation that need to be accounted for.
 
-Systematic biases, which can be induced by confounding variables, for example, can lead to artifacts or so-called "batch effects." 
+Systematic biases, which can be induced by confounding variables, for example, can lead to artifacts or so-called "batch effects."
 As a consequence, models may learn to rely on correlations that are irrelevant in the scientific context of the study and may result in misguided predictions and misleading conclusions [@doi:10.1038/nrg2825].
 Other study design considerations that should not be overlooked include knowing whether a study involves biological or technical replicates or both.
-For example, are some samples collected from the same individuals at different time points? 
-Are those time points before and after some treatment? 
+For example, are some samples collected from the same individuals at different time points?
+Are those time points before and after some treatment?
 If one assumes that all the samples are independent but that is in fact not the case, a variety of issues may arise, including having a lower effective sample size than expected.
 
 In general, deep learning has an increased tendency for overfitting, compared to classical methods, due to the large number of parameters being estimated, making issues of adequate sample size even more important (see [Tip 7](#overfitting)).
 For a large dataset, overfitting may not be a concern, but the modeling power of deep learning may lead to more spurious correlations and thus incorrect interpretation of results (see [Tip 9](#interpretation)).
 Finally, it is important to note that with the exception of very specific cases of unsupervised data analysis, it is generally the case that a molecular or imaging dataset does not have much value without appropriate clinical or demographic data; this must always be balanced with the need to protect patient privacy (see [Tip 10](#privacy)).
-Looking at these data can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.
+Looking at that data can also clarify the study design (for example, by seeing if all the individuals are adolescents or women) or at least help the analyst employing deep learning to know what questions to ask.

```

Comment:
Can we replace `that data` with `these annotations` to avoid the is/are debate here?